components/auth/org.wso2.carbon.auth.oauth/src/main/java/org/wso2/carbon/auth/oauth/dao/OAuthDAO.java

```diff
@@ -20,36 +20,37 @@
 
 package org.wso2.carbon.auth.oauth.dao;
 
-import org.wso2.carbon.auth.oauth.exception.ClientDAOException;
+import org.wso2.carbon.auth.oauth.dto.AccessTokenData;
+import org.wso2.carbon.auth.oauth.exception.OAuthDAOException;
 
 import java.net.URI;
 import java.util.Optional;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
 /**
- * DAO Interface to Client related data
+ * DAO Interface to OAuth related data
  */
-public interface ClientDAO {
+public interface OAuthDAO {
     /**
-     * Get public information related to a client including the clientId and redirectUri
+     * Get pre-registered redirectUri of a client
      *
      * @param clientId Client Id of client
      * @return Redirect Uri if exists else Optional.Empty
-     * @throws ClientDAOException if a DOA Error is encountered
+     * @throws OAuthDAOException if a DOA Error is encountered
```

Comment:
Spotted a small error in the comments. DOA should be changed to DAO. Shall we fix it in other places in the same class?